Drivers/ManagePersonsPermissionCheckerTaskDisplayDriver.cs

```diff
@@ -5,17 +5,20 @@
 
 namespace Lombiq.TrainingDemo.Drivers;
 
-// ActivityDisplayDriver is specifically for implementing Workflow Activity Tasks.
-// Don't forget to register this class with the service provider (see: Startup.cs).
-public class ManagePersonsPermissionCheckerTaskDisplayDriver :
-    ActivityDisplayDriver<ManagePersonsPermissionCheckerTask, ManagePersonsPermissionCheckerTaskViewModel>
+// ActivityDisplayDriver is specifically for implementing workflow tasks. It performs a simple mapping of a ManagePersonsPermissionCheckerTask
+// to a ManagePersonsPermissionCheckerTaskViewModel and vice versa. Don't forget to register this class with the service provider (see: Startup.cs).
```

Comment:
Break comments at 120 characters. See https://lombiq.atlassian.net/wiki/spaces/DEV/pages/28213305/General+code+conventions+and+development+practices#:~:text=For%20legibility%20try%20to%20limit%20lines%20of%20code%20to%20120%20characters also for how to automate this.